src/fragments/03-core-theme-light.scss

```diff
@@ -1,6 +1,19 @@
-.theme-dark .print,
```

Comment:
Before removing this, can you verify what happens when you print when you're in dark mode? Last I checked, the results were a little off.. So I basically flipped dark-mode to light mode if print was involved..